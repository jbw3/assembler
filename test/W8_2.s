MOV R1 # comment after instruction
# comment on its own line

# ^ blank line
ADD R2
MOV R7

# indentation
    MOV R1
    ADD R2
    MVR0 R1


# immediate values
MOVI -4
MOVI -3
MOVI -2
MOVI -1
MOVI 0
MOVI 1
MOVI 2
MOVI 3
