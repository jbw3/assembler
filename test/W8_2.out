09
42
0f
09
42
19
10
11
12
13
14
15
16
17
